src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -140,4 +141,37 @@ case class Run(runId: String, pipeline: Pipeline, logger: WorkflowLogger) {
     val cancellationRequest: CancelOperationRequest = new CancelOperationRequest()
     pipeline.genomicsService.operations().cancel(runId, cancellationRequest).execute
   }
+
+  private def getEventList(op: Operation): Seq[ExecutionEventEntry] = {
+
+    // An execution event entry with only a "startTime"
+    case class HalfJesEvent(name: String, timestamp: DateTime)
+
+    val createEvent = Seq(HalfJesEvent("chilling out, maxing", DateTime.parse(op.getMetadata.get("createTime").asInstanceOf[String])))
+    val startEvent = Seq(HalfJesEvent("relaxing all cool", DateTime.parse(op.getMetadata.get("startTime").asInstanceOf[String])))
```

Comment:
are we guaranteed all these metadata events will be defined for runs that exit abnormally?